cms/djangoapps/contentstore/views/course.py

```diff
@@ -704,17 +703,11 @@ def format_course_for_view(course):
         """
         Return a dict of the data which the view requires for each course
         """
-        if isinstance(course.id, SlashSeparatedCourseKey):
-            course_module = modulestore().get_course(course.id, depth=0)
-            lms_link = get_lms_link_for_item(course_module.location)
-        else:
-            lms_link = get_lms_link_for_item(course.location)
-
         return {
             'display_name': course.display_name,
             'course_key': unicode(course.location.course_key),
             'url': reverse_course_url('course_handler', course.id),
-            'lms_link': lms_link,
+            'lms_link': get_lms_link_for_item(course.location),
```

Comment:
We had forked all of this for a bugfix some time back (@dcadams you, I think?),
so this is an unfork.